graylog2-web-interface/src/components/common/NoSearchResult.tsx

```diff
@@ -18,24 +18,29 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 
 import { Alert } from 'components/bootstrap';
+import { Icon } from 'components/common';
 
 type Props = {
   children: React.ReactNode,
   className: string | null | undefined,
```

Comment:
Not related to this PR, but let's use `className?: string` here as well. We used `string | null | undefined` when we migrated from flow to TS to ensure we use the exact same type definition.